requirements.txt

```diff
@@ -1,4 +1,4 @@
 matplotlib
-numpy>=1.8
+numpy
 pandas
-pandas>=1.4;python_version>="3.8"
```

Comment:
Pandas minimum version was for the compare dataframe method (which I think you also have in the tests?)